docs/agented-vm-guide.md

```diff
@@ -0,0 +1,121 @@
+# Agented VM User Guide
+
+### Antrea User
+
+For `antrea-agent` configuration and installation, please refer to
+antrea [ExternalNode](https://github.com/antrea-io/antrea/blob/main/docs/external-node.md#install-antrea-agent-on-vm)
+documentation.
+
+Note: While applying antrea manifests, make sure you enable `externalnode`
+feature in the `antrea-controller` configuration. Also make sure that VM
+can connect to antrea API server
+
+### Nephe User
```

Comment:
Please make doc in reviewable state

Reply:
I will add documentation in a later commit. Want to get the functionality changes in.